nutils/evaluable.py

```diff
@@ -422,12 +422,10 @@ def eval(**args):
             log.info('total time: {:.0f}ms\n'.format(tottime/1e6) + '\n'.join('{:4.0f} {} ({} calls, avg {:.3f} per call)'.format(t / 1e6, k, n, t / (1e6*n))
                                                                               for k, t, n in sorted(aggstats, reverse=True, key=lambda item: item[1]) if n))
 
-    def _stack(self, values):
-        lines = ['  %0 = EVALARGS']
-        for (op, indices), v in zip(self.serialized, values):
-            lines[-1] += ' --> ' + type(v).__name__
-            if numeric.isarray(v):
-                lines[-1] += '({})'.format(','.join(map(str, v.shape)))
+    @property
```

Comment:
I'm not sure if a property makes sense for iterators. It's not immediately clear why you would get a new iterator every time you access the `_iter_stack` attribute for some instance of `Evaluable`.

Reply:
true, fixed